notebooks/name-entity-extraction.ipynb

```diff
@@ -0,0 +1,785 @@
+{
```

Comment:
<p>Move this before "Lets load"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/optimum-graphcore-fork/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>